JitsiConference.js

```diff
@@ -552,24 +500,98 @@ JitsiConference.prototype.removeTrack = function (track) {
         return Promise.reject(
             new JitsiTrackError(JitsiTrackErrors.TRACK_IS_DISPOSED));
     }
+    return this.replaceStream (track, null);
+};
 
-    if(!this.room){
-        if(this.rtc) {
-            this.onTrackRemoved(track);
+/**
+ * Replaces oldStream with newStream and performs a single offer/answer
+ *  cycle after both operations are done.  Either oldStream or newStream
+ *  can be null; replacing a valid 'oldStream' with a null 'newStream'
+ *  effectively just removes 'oldStream'
+ * @param oldStream the current stream in use to be replaced
+ * @param newStream the new stream to use
+ * @returns {Promise}
+ */
+JitsiConference.prototype.replaceStream = function (oldStream, newStream) {
```

Comment:
there is quite a bit of inconsistency in the usage of track vs. stream.  i started with 'replaceTrack' but all the methods beneath it that are called are named with 'stream'.

fixed the disposed checks